src/components/UI/DropDownMenu/ItemDropDown.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <Dropdown
+    v-model="localSelectedItemIndex"
+    :options="optionsList"
+    optionLabel="label"
+    optionValue="index"
+    placeholder="dummy placeholder"
```

Comment:
would be better to not add any placeholder? or putting an appropriate text here?